docs/mine/lotus/miner-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Lotus Miner: lifecycle'
+description: 'How to safely perform maintenance on a Lotus miner.'
+breadcrumb: Lifecycle'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+These operations are normally related to maintenances and upgrades. Given that miners are expected to submit proofs to the chain in a continuous fashion while running long and expensive operations, it is important that operators become familiar with how to manage some of the events in the miner lifecycle so that they can be performed with the maximum guarantees.
+
+[[TOC]]
+
+## Safely restarting the miner daemon
+
+Before shutting down your miner daemon, make sure that there are no pending operations that could get your miner _slashed_.
```

Comment:
```suggestion
Before shutting down your miner daemon, make sure that there are no pending operations that could get your miner _slashed_ if they are delayed or cancelled.
```

Reply:
This sentence should point to a section on how to check the mpool for pending messages.

Reply:
Isn't enough with the `proving info` below (or `proving dearlines`)?

Reply:
> how to check the mpool for pending messages.

Can you actually tell me how?

Reply:
Another question is how to make a miner stop taking deals without restarting it (so that we can be sure it has not just closed another deal and it's receiving data when its stopped).

Reply:
Actually, yes to restart just the miner checking proving is sufficient. But in case here is how you check pending messages, `lotus mpool pending --local | jq -c . | wc -l` will give you the number of pending messages. To stop taking deals you can do `lotus-miner storage-deals selection reject --online --offline` and `lotus-miner retrieval-deals selection reject --online --offline`